subprojects/docs/src/docs/release/notes.md

```diff
@@ -82,96 +82,102 @@ vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
 #### Added support for incremental compilation following a compilation failure
 
-Gradle already supports [Java incremental compilation](userguide/java_plugin.html#sec:incremental_compile) by default and [Groovy incremental compilation](userguide/groovy_plugin.html#sec:incremental_groovy_compilation) as an opt-in experimental feature.
-
-In previous versions after a compilation failure the next compilation was not incremental but a full recompilation instead.
-
-With this version, Java and Groovy incremental compilation will work incrementally also after a failure.
-
-This improves experience with compilation when working iteratively on some Java or Groovy code, e.g. when iteratively running compile or test tasks from an IDE.
+Gradle supports [Java incremental compilation](userguide/java_plugin.html#sec:incremental_compile)
+by default and [Groovy incremental compilation](userguide/groovy_plugin.html#sec:incremental_groovy_compilation)
+as an opt-in experimental feature.
 
+In previous versions, a compilation failure caused the next compilation to fully recompile.
+In Gradle 7.6 and above, Java and Groovy incremental compilation works even after a failure.
```

Comment:
Hmm is it more correct to say “From Gradle 7.6 and above”?

Reply:
Or just “From Gradle 7.6”

Reply:
"Starting in Gradle 7.6" is the better pattern I've seen -- I'll switch to that.